src/ParsePushAdapter.js

```diff
@@ -49,8 +51,19 @@ export class ParsePushAdapter {
     let deviceMap = classifyInstallations(installations, this.validPushTypes);
     let sendPromises = [];
     for (let pushType in deviceMap) {
-      let sender = this.senderMap[pushType];
+      var sender = this.senderMap[pushType];
       let devices = deviceMap[pushType];
+      
+      if (!sender) {
+        if (pushType == 'tvos') {
```

Comment:
not sure this fallback is required. Even if it leads to redundancy for all 'apple' push types.